Common/include/CConfig.hpp

```diff
@@ -5135,21 +5135,19 @@ class CConfig {
   bool GetHold_GridFixed(void) const { return Hold_GridFixed; }
 
   /*!
+   * \author H. Kline
    * \brief Get the kind of objective function. There are several options: Drag coefficient,
    *        Lift coefficient, efficiency, etc.
    * \note The objective function will determine the boundary condition of the adjoint problem.
+   * \param[in] val_obj
    * \return Kind of objective function.
    */
-  unsigned short GetKind_ObjFunc(void) const { return Kind_ObjFunc[0]; }
+  unsigned short GetKind_ObjFunc(unsigned short val_obj = 0) const { return Kind_ObjFunc[val_obj]; }
 
   /*!
-   * \author H. Kline
-   * \brief Get the kind of objective function. There are several options: Drag coefficient,
-   *        Lift coefficient, efficiency, etc.
-   * \note The objective function will determine the boundary condition of the adjoint problem.
-   * \return Kind of objective function.
+   * \brief Similar to GetKind_ObjFunc but returns the corresponding string.
    */
-  unsigned short GetKind_ObjFunc(unsigned short val_obj) const { return Kind_ObjFunc[val_obj]; }
+  string GetName_ObjFunc(unsigned short val_obj = 0) const;
```

Comment:
for completeness can we get the param[in] and return here as well

Reply:
sorry I just find that kind of description too much fluff, one of my pet peeves with this doxy stuff is that a ton of methods describe the same arguments over and over ad nauseam and they say very little of the actual method. Less is more is what I'm getting at.

Reply:
For example:
`* \brief  A virtual member.`
"No **** Sherlock, I could see that from just the code..." The important is what the method should do.
Anyway, rant over.